web/public/robots.txt

```diff
@@ -1 +1,3 @@
-User-agent: *
\ No newline at end of file
+User-agent: *
+
+Sitemap: http://app.electricitymaps.com/sitemap.xml
```

Comment:
This is valid syntax and recommended by google: https://developers.google.com/search/docs/crawling-indexing/robots/create-robots-txt